engine/settings/base.py

```diff
@@ -348,6 +348,10 @@ class DatabaseTypes:
     "/alertgroups",
     "/alert_receive_channels",
     "/webhooks",
+    "/channel_filters",
+    "/escalation_chains",
+    "/escalation_policies",
+    "/notification_policies",
```

Comment:
might be a good idea to re-generate frontend types and commit them in this PR as well